torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -1199,6 +1199,21 @@ def reshape_1d(code, args=args, kwargs=kwargs):
                 cast_to_int32 = g.call_function(ttnn.typecast, args=(args[0], TtnnInt32()))
                 return g.call_function(ttnn.bitwise_not, args=(cast_to_int32,))
 
+            if node.target == torch.ops.aten.linalg_vector_norm.default:
+                x, ord, dim, keepdim = args
+
+                try:
+                    ndims = len(x.meta["val"].size())
+                    dim = [d if d >= 0 else d + ndims for d in dim]
+                except:
```

Comment:
what exception does this cause?

Reply:
If the shape of `x` is not known at compile time, `x.meta["val"].size()` fails.

Reply:
maybe then better to handle inside the target wrapper where values are already known?

Reply:
I change it to more graceful `get_shape`